Source/Classes/AnimatedImages/PINWebPAnimatedImage.m

```diff
@@ -278,6 +385,38 @@ - (CGImageRef)imageAtIndex:(NSUInteger)index cacheProvider:(nullable id<PINCache
     return canvas;
 }
 
+- (BOOL)frameRequiresBlendingWithPreviousFrame:(NSUInteger)index
```

Comment:
This seems very similar to the isKeyFrame logic… can that just be updated instead?